automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -12,12 +11,18 @@ import org.scalatest.selenium.Page
 class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodConfigNamespace: String, val methodConfigName: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceMethodConfigDetailsPage] {
 
+  private val tab = TabBar()
+  private val methodConfigListpage = new WorkspaceMethodConfigListPage(namespace, name)
+
   override def awaitReady(): Unit = {
-    await condition isLoaded
-    await spinner "Checking permissions..."
+    await condition {
+      tab.isTabOpen("Method Configurations")
```

Comment:
We don't need to be testing the state of the tab bar. This page class shouldn't know anything about the tab bar, that's WorkspacePage's responsibility.